README.rst

```diff
@@ -330,6 +319,19 @@ additional URL from the configuration file:
    ID              Version Name
    esp8266:esp8266 2.5.2   esp8266
 
+Alternatively you can pass a link to the the additional package index file with the ``--additional-urls`` option to
```

Comment:
we could stress the fact that you always have to pass the option if you decide to not use the config file, something on these lines:

> Alternatively, you can pass a link to the the additional package index file with the `--additional-urls` option, but this has to be done every time and for every command that operates on a platform core, for example:

Reply:
:+1: I have rephrased it a bit, please check if it makes sense.